types/duckduckgo-images-api/index.d.ts

```diff
@@ -0,0 +1,32 @@
+// Type definitions for duckduckgo-images-api 1.0
+// Project: https://github.com/KshitijMhatre/duckduckgo-images-api
+// Definitions by: Bart Duisters <https://github.com/bartduisters>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function image_search(params: {
+    query: string,
+    moderate: boolean,
+    retries: number,
+    iterations: number
+}): Promise<Array<{
+    image: string,
+    title: string,
+    height: number,
+    thumbnail: string,
+    width: number,
+    url: string,
+    source: string}>>;
+
+export function image_search_generator(params: {
+    query: string,
```

Comment:
nit: Both `params` and the returned properties are the same for `image_search` and `image_search_generator`. You could extract that to its own type and reuse them on both functions. It will make it easier to read as well.

Reply:
Extracted to an interface.